src/main/resources/templates/athlete.html

```diff
@@ -1,41 +1,42 @@
 <!DOCTYPE html>
 <html>
+
 <head>
-<meta charset="UTF-8"></meta>
-<link rel="stylesheet" type="text/css" media="screen" href="style.css"/>
-<title>Athlete Workout Page</title>
+	<meta charset="UTF-8">
+	</meta>
+	<link rel="stylesheet" type="text/css" media="screen" href="style.css" />
+	<title>Athlete Workout Page</title>
 </head>
+
 <body>
-<h1>Athlete Login Page</h1>
-<a href="index">
-	<button>Home</button>
-</a>
-</br>
-</br>
+	<h1>Athlete Login Page</h1>
 
-View Your Scheduled Workouts
-</br>
-</br>
+	<a href="index">
+		<button>Home</button>
+	</a>
 
-<table>
-	<tr>
-		<th>Monday</th>
-		<th>Tuesday</th>
-		<th>Wednesday</th>
-		<th>Thursday</th>
-		<th>Friday</th>
-		<th>Saturday</th>
-		<th>Sunday</th>
-	</tr>
-	<tr>
-		<th height="600"></th>
-		<th></th>
-		<th></th>
-		<th></th>
-		<th></th>
-		<th></th>
-		<th></th>
-	</tr>
-</table>
+	<h3>View Your Scheduled Workouts</h3>
+
+	<table>
+		<tr>
+			<th>Monday</th>
+			<th>Tuesday</th>
+			<th>Wednesday</th>
+			<th>Thursday</th>
+			<th>Friday</th>
+			<th>Saturday</th>
+			<th>Sunday</th>
+		</tr>
+		<tr>
+			<th class="height-600"></th>
+			<th></th>
```

Comment:
These probably need to be td tags.  th is usually only used on the very first row, for the table header.